dpdata/plugins/deepmd.py

```diff
@@ -69,6 +71,88 @@ def from_labeled_system(self, file_name, type_map=None, **kwargs):
     MultiMode = Format.MultiModes.Directory
 
 
+@Format.register("deepmd/mixed")
+class DeePMDMixedFormat(Format):
+    def from_system_mix(self, file_name, type_map=None, **kwargs):
+        return dpdata.deepmd.mixed.to_system_data(
+            file_name, type_map=type_map, labels=False
+        )
+
+    def to_system(self, data, file_name, set_size=200, prec=np.float64, **kwargs):
```

Comment:
Is there any reason to set the default of set size to 200? If only one set is supported, I think we should remove this option.